.circleci/config.yml

```diff
@@ -301,6 +399,15 @@ workflows:
           filters:
             branches:
               only: /^dependabot/submodules/.*/
+      - android-build-and-publish-react-native-bridge:
+          name: Build Android RN Bridge & Publish to Bintray
+          filters:
+            branches:
+              # Don't run on trunk, otherwise, run on every other branch
+              ignore: trunk
+            tags:
+              # Run for every tag
+              only: /.*/
```

Comment:
Here's an example of build from a tag: [`mokagio-test-20210317-173215`](https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/10980/workflows/2b2eaf57-d638-42ce-80bf-def2720ba820/jobs/57750).